epsagon/modules/tornado.py

```diff
@@ -34,6 +37,15 @@ def before_request(cls, wrapped, instance, args, kwargs):
             if not ignored:
                 cls.RUNNER = TornadoRunner(time.time(), instance.request)
                 trace.set_runner(cls.RUNNER)
+
+                # Collect metadata in case this is a container.
+                if not cls.CHECKED_ECS_ENDPOINT:
```

Comment:
just have the collect_container_metatdata() function cache it (can be a singleton) instead of caching it in every implementation

Reply:
Fixed